strings/strings.c

```diff
@@ -20,7 +26,16 @@ int string_length(char s[])
 */
 char *reverse_string(char rv[], char s[])
 {
-
+    int length = string_length(s):
+    int index = 0;
+    for (int i = length - 1; i >= 0; i--)
+    {
+        rv[index] = s[i];
+        index ++;
+
+    }
+    rv[length] = '\0'
```

Comment:
Missing a semi-colon here:
```suggestion
    rv[length] = '\0';
```